SparkPost/2web2ui
# FE-932: Skipped timezone test

### What Changed
 - Skipped unit test for timezone typeahead

### How To Test
 - Run timezone typeahead test
 - Verify it passes
